ESP32LapTimer/src/ESP32LapTimer.cpp

```diff
@@ -74,7 +69,17 @@ void setup() {
   //PowerDownAll(); // Powers down all RX5808's
   delay(250);
 
+#if defined(WIFI_MODE_AP)
   InitWifiAP();
+#elif defined(WIFI_MODE_CLIENT)
```

Comment:
I think it is okay for now, but in the long run a runtime config would be nice

Reply:
It would be nice to have, agreed, but i think ill leave that for a separate pull request